File: Bronze5/27433.js
// const input = require("fs").readFileSync("/dev/stdin").toString();
const input = "1".toString();

function factorial2(input) {
  const N = Number(input);
  let result = 1;

  for (let i = 1; i <= N; i++) {
    result *= i;
  }

  return result;
}

console.log(factorial2(input));

File: Bronze5/27433.py
# N = int(input())

# if N == 0:
#     print(1)
# else:
#     result = 1
#     for i in range(2, N + 1):
#         result *= i
#     print(result)

import sys, math

print(math.factorial(int(sys.stdin.readline())))